client/src/components/VideoDisplay.js

```diff
@@ -0,0 +1,101 @@
+import { useState } from "react";
+import AddVideo from "./AddVideo";
+import SearchVideo from "./SearchVideo"
+
+const VideoDisplay = (prop) => {
+  const [allvideos, setAllVideos] = useState(prop.video);
+  const [searched, setsearchedVideos] = useState(prop.video);
+  const inputVideo = (newvideo) => {
+    setAllVideos([...allvideos].concat(newvideo));
```

Comment:
you could also do `setAllVideos([...allVideos, newVideo])` here :)

Reply:
Hi Naomi, I tried setAllVideos([...allVideos, newVideo]) but for some reason, this is not working and giving an
 error instead :(

Reply:
Hi Anza, sorry I should have expanded here, your newVideo will need to be an object for `([...allVideos, newVideo])`to work (I believe it's an object in an array at the moment?)